template_migrations/20230423094420_webperf_rum_events.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS {prefix}webperf_rum_events;
```

Comment:
Is it possible to have migrations that a numbered like:
- 1_webperf_rum_events.down.sql 
- 2_webperf_rum_events.down.sql 
- 3_webperf_rum_events.down.sql 
 
 Instead of 20230423094420_webperf_rum_events.down.sql

Reply:
is this ok 00000000000001_webperf_rum_events.down.sql ?